package.json

```diff
@@ -1,10 +1,18 @@
 {
-  "dependencies": {
-    "docx": "^7.7.0",
-    "pdf-to-text": "^0.0.7"
-  },
-  "devDependencies": {
-    "@types/node": "^18.11.10",
-    "typescript": "^4.9.3"
-  }
+   "dependencies": {
+      "docx": "^7.7.0",
+      "pdf-to-text": "^0.0.7"
+   },
+   "devDependencies": {
+      "@types/node": "^18.11.10",
+      "@typescript-eslint/eslint-plugin": "^5.45.0",
+      "@typescript-eslint/parser": "^5.45.0",
+      "eslint": "^8.29.0",
+      "prettier": "2.8.0",
+      "typescript": "^4.9.3"
+   },
+   "scripts": {
```

Comment:
Use scripts to quickly execute commands. For example now you can lint your repo with `npm run lint` and prettify your repo with `npm run prettify`